modelci/cli/__init__.py

```diff
@@ -22,16 +22,20 @@ def cli():
     pass
 
 
-@cli.command()
-@click.option('--gpu', default=False, type=click.BOOL, is_flag=True)
-def start(gpu=False):
+def __start__(gpu=False):
```

Comment:
Why use this __xxx__ format to name the function?

We should not use this style in case of having conflicts with python build-in functions

Reply:
@YuanmingLeee  Enabling gpu here is not appropriate. Even the leader server has no GPUs, it should run our modelci with ease. Users do not need to enable GPU when starting mlmodelci.
I think only the profiler and dispatcher need GPU Dockers when necessary.

Reply:
yes, but this `modelci start` command will start the required Docker images for profiler and dispatcher. Those Docker images should be specify w GPU or w/o GPU.